Formula/kmdr-cli.rb

```diff
@@ -0,0 +1,19 @@
+require "language/node"
+
+class KmdrCli < Formula
+  desc "CLI for explaining shell commands"
+  homepage "https://kmdr.sh/"
+  url "https://registry.npmjs.org/kmdr/-/kmdr-0.1.34.tgz"
+  sha256 "233814fb6665470720468ef419ff17c7fddcc47182785f979149352adfae30cf"
+  head "https://github.com/ediardo/kmdr-cli.git"
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system "true"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.